app/templates/Articles/index.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @var \App\View\AppView $this
+ * @var iterable<\Cake\Datasource\EntityInterface> $articles
+ */
+?>
+<div class="articles index content">
+    <?= $this->Html->link(__('New Article'), ['action' => 'add'], ['class' => 'button float-right']) ?>
+    <h3><?= __('Articles') ?></h3>
+    <div class="table-responsive">
+        <table>
+            <thead>
+                <tr>
+                    <th><?= $this->Paginator->sort('id') ?></th>
+                    <th><?= $this->Paginator->sort('user_id') ?></th>
+                    <th><?= $this->Paginator->sort('title') ?></th>
+                    <th><?= $this->Paginator->sort('published') ?></th>
+                    <th><?= $this->Paginator->sort('created') ?></th>
+                    <th><?= $this->Paginator->sort('modified') ?></th>
+                    <th class="actions"><?= __('Actions') ?></th>
+                </tr>
+            </thead>
+            <tbody>
+                <?php foreach ($articles as $article): ?>
+                <tr>
+                    <td><?= $this->Number->format($article->id) ?></td>
+                    <td><?= $this->Number->format($article->user_id) ?></td>
+                    <td><?= h($article->title) ?></td>
+                    <td><?= h($article->published) ?></td>
+                    <td><?= h($article->created) ?></td>
+                    <td><?= h($article->modified) ?></td>
```

Comment:
Both created & modified is an instance of \Cake\I18n\FrozenTime. Refer the topic related to Date & Time https://book.cakephp.org/4/en/core-libraries/time.html#namespace-Cake\I18n

Since we passed the object FrozenTime as the argument for h() function, it'll be converted to string at the first place. Refer the magic method __toString defined for FrozenTime/FrozenDate  https://github.com/cakephp/cakephp/blob/86f6340e4b958ddacbee8d62dd5e9bd8f9b6be1a/src/I18n/DateFormatTrait.php#L268-L274

Calling the method i18nFormat() without argument caused the formatted result become locale dependent based on locale configuration (check `config/app.php`). Refer https://book.cakephp.org/4/en/core-libraries/internationalization-and-localization.html#setting-the-default-locale

Try to change the locale to `es_ES`, `ja_JP`, etc. We can see the output itself will vary for each locale. 

Normally,  we'll confirm the specific formatting to display the date and time. We've to be very careful with the instance of FrozenTime/FrozenDate.

Reply:
For displaying date & time, can we display it in ISO8601 format? Please kindly check.